egs/csj/asr1/run.sh

```diff
@@ -195,9 +212,9 @@ mkdir -p ${expdir}
 
 if [ ${stage} -le 4 ]; then
     echo "stage 4: Network Training"
-    ${cuda_cmd} ${expdir}/train.log \
+    ${cuda_cmd} --gpu ${ngpu} ${expdir}/train.log \
```

Comment:
Just make sure that do we need some changes in `cmd.sh` to pass `--gpu ${ngpu}`?

Reply:
In the case of slurm.pl, we do not have to modify `cmd.sh`.